.github/workflows/coverage.yml

```diff
@@ -67,12 +70,20 @@ jobs:
       - name: Install binutils and rustfilt
         run: cargo install cargo-binutils rustfilt
 
+      - name: Build for tests
+        uses: actions-rs/cargo@v1
+        env:
+          RUSTFLAGS: "-C instrument-coverage"
+          LLVM_PROFILE_FILE: "iota-%m.profraw"
```

Comment:
Are they needed here? Isn't it enough if they get set in coverage.sh?

Reply:
They are. Without them, it would rebuild again in the script.